tech/post/The details of share network.md

```diff
@@ -0,0 +1,217 @@
+# What's the details of share network in manila
+
+# What's the DHSS in manila
+
+OpenStack Manila provides secure filesystem storage over the network.
+In multi-tenant environments, it is expected that file shares are
+isolated between different tenants in the data path. Achieving strong
+guarantees for data path isolation requires back end support, and cloud
+administrators can configure their deployments to provide such isolation.
+We create a option named "driver_handles_share_servers" (DHSS) which
+defines the mode that back end drivers operate.
+
+# Create share with DHSS CLI
+
+https://docs.openstack.org/mitaka/install-guide-ubuntu/launch-instance-manila-dhss-true-option2.html
+
+
+# Enviroment
+My openstack enviroment is installed by devstack and use generic driver
+with private network.
+
+
+Config in generic
+
+[generic1]
+admin_subnet_id = 2c745913-7766-45b3-abfc-33a527e5185b
+admin_network_id = dd07dbd2-cb02-4ef3-a8c0-5b3e75013c92
+driver_handles_share_servers = True
+service_instance_user = manila
+service_password = manila
+service_image_name = manila-service-image-master
+path_to_private_key = /opt/stack/.ssh/id_rsa
+path_to_public_key = /opt/stack/.ssh/id_rsa.pub
+share_backend_name = GENERIC1
+share_driver = manila.share.drivers.generic.GenericShareDriver
+
+
+# Details about network
+
+In creating the share workflow. We are going to create a instance as
+the infrastructure for share server. The share server used for isolated the
+network of the share. After the share verver has been created, we are going
+to mount a cinder volume to that instance, the cinder volume which is going
+to be created by the manila, and export the volume. After that the user can
+directly use the path of the volume mount to any other instances which has the
+ability to access the share.
+
+
+1. Get service image
+
+We are going to get the image object by glance image list api. The
+image name be got from generic driver option named ``service_image_name``.
```

Comment:
How about this? The image name is defined in generci driver option: `service_image_name`